docs/userguides/userriskprofile.md

```diff
@@ -0,0 +1,68 @@
+# User Risk Profile
+
+A user risk profile is created for each user.  Use py42 to manage these user risk profiles.
+
+## Update a User Risk Profile
+
+Determine the user ID to manage a user's risk profile.  For example, the following code uses the `get_all()` method to look for the ID of a user with the username `test.user@code42.com`.
```

Comment:
We should probably just turn this code into a helper method, something like `sdk.userriskprofile.get_by_username()` and rename `.get()` to `.get_by_id()`.

Reply:
added !